### PR TITLE
fix: Minor Bug Fixes

### PR DESCRIPTION
**Pull request by thetechie7**
1. There was an extra `rounded` class added to scan page, removed that.
2. Tailwind was showing warnings like:
```
warn - The `purge`/`content` options have changed in Tailwind CSS v3.0.
warn - Update your configuration file to eliminate this warning.
warn - https://tailwindcss.com/docs/upgrade-guide#configure-content-sources

warn - The `darkMode` option in your Tailwind CSS configuration is set to `false`, which now behaves the same as `media`.
warn - Change `darkMode` to `media` or remove it entirely.
warn - https://tailwindcss.com/docs/upgrade-guide#remove-dark-mode-configuration
```
Upadated the config file to `content` instead of `purge` and disabled `darkMode`
3. The Challenges were not showing in the admin page because the props were made for the `props` not for `props.challenges`, fixed that by passing the write props.
4. Speaker card Images were streched down, fixed that adding `objectFit="cover"`. The cards were also missing a subtitle as per kelly's design added them, and updated the styling for the ` title ` and ` subtitle `.

### CODE REVIEW

File: components/homeComponents/KeynoteSpeaker.tsx [hunk 1]
```diff
@@ -32,14 +32,16 @@ export default function KeynoteSpeaker(props) {
...
             alt=""
           />
         )}
       </div>
-      <div className="flex-col items-center justify-center w-full h-72 sm:h-80 absolute translate-y-52 sm:translate-y-64 group-hover:translate-y-0 transition duration-500 ease-in-out overflow-hidden">
-        <div className="rounded-b-sm bg-primary p-4 font-semibold">
-          <h1 className="text-base font-medium"> {props.name}</h1>
+      <div className="flex-col items-center justify-center w-full h-72 sm:h-80 absolute translate-y-56 sm:translate-y-64 group-hover:translate-y-0 transition duration-500 ease-in-out overflow-hidden">
+        <div className="rounded-b-sm bg-primary p-2 font-semibold">
+          <h1 className="text-lg font-medium"> {props.name}</h1>
```
Comment (NickBurnett) [inline components/homeComponents/KeynoteSpeaker.tsx:43]: Are the prefix spaces necessary?

File: components/adminComponents/challengeComponents/ChallengeList.tsx [hunk 2]
```diff
@@ -54,10 +54,12 @@ export default function ChallengeList({
     <div className="p-5">
       <DndContext sensors={sensors} collisionDetection={closestCenter} onDragEnd={handleDragEnd}>
         <SortableContext items={challenges} strategy={verticalListSortingStrategy}>
-          {challenges.map((challenge, idx) => (
+          {challenges.map(({ title, description, prizes }, idx) => (
             <ChallengeCard
-              challenge={challenge}
-              key={challenge.id}
+              key={idx}
+              title={title}
```
Comment (NickBurnett) [inline components/adminComponents/challengeComponents/ChallengeList.tsx:60]: Not a fan of us arbitrarily adding props when we haven't defined any for the component.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   mode: 'jit',
-  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
-  darkMode: false, // or 'media' or 'class'
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
+  // darkMode: 'class',
```
Comment (NickBurnett) [inline tailwind.config.js:4]: We need to look into dark mode later on, not urgent though.

**Review by NickBurnett — CHANGES_REQUESTED** [review]
A few things, a lot of it isn't your fault but it'd be great if you could fix it if the fixes are urgent. Also, in the future please give a description of your PR so we know what's going on at a high level.

File: components/homeComponents/KeynoteSpeaker.tsx [hunk 4]
```diff
@@ -32,14 +32,16 @@ export default function KeynoteSpeaker(props) {
...
+            height={350}
+            objectFit="cover"
             alt=""
           />
         )}
       </div>
-      <div className="flex-col items-center justify-center w-full h-72 sm:h-80 absolute translate-y-52 sm:translate-y-64 group-hover:translate-y-0 transition duration-500 ease-in-out overflow-hidden">
-        <div className="rounded-b-sm bg-primary p-4 font-semibold">
-          <h1 className="text-base font-medium"> {props.name}</h1>
+      <div className="flex-col items-center justify-center w-full h-72 sm:h-80 absolute translate-y-56 sm:translate-y-64 group-hover:translate-y-0 transition duration-500 ease-in-out overflow-hidden">
```
[the remark this answers is not in the document]
Comment (thetechie7) [reply components/homeComponents/KeynoteSpeaker.tsx:41]: just wrote one above

File: components/adminComponents/challengeComponents/ChallengeList.tsx (hunk 2 above)
Comment (thetechie7) [reply components/adminComponents/challengeComponents/ChallengeList.tsx]: removed it

File: tailwind.config.js (hunk 3 above)
Comment (thetechie7) [reply tailwind.config.js:4]: ok

File: components/homeComponents/KeynoteSpeaker.tsx (hunk 1 above)
Comment (thetechie7) [reply components/homeComponents/KeynoteSpeaker.tsx]: removed it

**Comment by MichaelZhao21** [comment]
geeth why u requesting reviews on your DRAFT PR

**Comment by thetechie7** [comment]
> geeth why u requesting reviews on your DRAFT PR
i been screwing up a lot myb

**Comment by MichaelZhao21** [comment]
its okay geeth ur doing great

**Comment by thetechie7** [comment]
im def getting pipped...

**Comment by abdullah50053** [comment]
They're just haters Geeth do ur thing (But also do QA)

**Comment by NickBurnett** [comment]
You're fine man keep doing what you're doing 👍🏼

**Review by NickBurnett — APPROVED** [review]

**Review by MichaelZhao21 — APPROVED** [review]

**Pull request merged**